Change already including in Upstream master:

Fix arg 1 of pthread_setaffinity_np
prototype passes pthread_t, not *pthread_t

Signed-off-by: Neil Horman <nhorman@tuxdriver.com>

---

diff -aur a/rngd_jitter.c b/rngd_jitter.c
--- a/rngd_jitter.c
+++ b/rngd_jitter.c
@@ -83,7 +83,7 @@
 				start_routine, arg);
 
 #ifdef HAVE_PTHREAD_SETAFFINITY
-	pthread_setaffinity_np(&thread_ctx->notime_thread_id, cpusize, cpus);
+	pthread_setaffinity_np(thread_ctx->notime_thread_id, cpusize, cpus);
 #endif
 
 	CPU_FREE(cpus);
